fdbserver/include/fdbserver/DDTeamCollection.h

```diff
@@ -286,6 +286,8 @@ class DDTeamCollection : public ReferenceCounted<DDTeamCollection> {
 
 		double pivotAvailableSpaceRatio = 0.0;
 		double pivotCPU = 100.0;
+		double strictPivotAS = 0.0;
```

Comment:
It is okay to be a bit verbose here (AS -> AvailableSpace) to prevent confusion while reading.

Reply:
Done